test/OpenBCISample-test.js

```diff
@@ -55,9 +56,8 @@ describe('OpenBCISample',function() {
             //console.log(temp);
             var taco = new Buffer([0x81]);
             taco.copy(temp,2);
-            //console.log(temp);
-            var sample = OpenBCISample.convertPacketToSample(sampleBuf);
-            assert(sample.channelData[1], -8323071 * OpenBCISample.scaleFactorChannel);
+            var sample = openBCISample.convertPacketToSample(temp);
+            sample.channelData[0].should.be.approximately(-8323071 * openBCISample.scaleFactorChannel, 0.001);
```

Comment:
A comparison between chai should and assert? Or what the differences are between what I'm testing? The way this test is currently if something broke withr the conversion we would know here